### PR TITLE
ztp: readme cleanup

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: PROVISIONING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Discovery and Provisioning
 
-## Docs
+## Supportive Docs and alternative specs
```
Comment (mestery) [inline PROVISIONING.md:3]: Weird capitalization here. Should be `Supportive docs and alternative spec`.

File: PROVISIONING.md [hunk 2]
```diff
@@ -42,36 +42,36 @@
...
+## Adopted solution for RFC 8572 Secure Zero Touch
 
-tbd... re-celling, faults...
+See [ZTP](ZTP.md) for diagrams and detailed explanation.
 
-## Manual Provisioning
+See [Demo](https://github.com/opiproject/opi-prov-life/tree/main/examples/sztp) for running example.
 
-Andy: Do we need to cover manual methods in our scope? Many, many methods...
+## Additional Provisioning methods out of OPI scope
```
Comment (mestery) [inline PROVISIONING.md:51]: Same problem, should be `Additional provisioning methods out of OPI scope`.

File: PROVISIONING.md [hunk 3]
```diff
@@ -42,36 +42,36 @@
...
 
-### RSHIM custom Provisioning
+Some provisioning can be triggered by an external actors either manually or automatically.
```
Comment (mestery) [inline PROVISIONING.md:53]: Remove `an` here.

**Review by mestery — CHANGES_REQUESTED** [review]
Some cleanups required, but overall is nice!

File: PROVISIONING.md (hunk 1 above)
Comment (glimchb) [reply PROVISIONING.md:3]: fixed

File: PROVISIONING.md (hunk 2 above)
Comment (glimchb) [reply PROVISIONING.md:51]: fixed

File: PROVISIONING.md (hunk 3 above)
Comment (glimchb) [reply PROVISIONING.md:53]: done

**Review by mestery — APPROVED** [review]
Lovely, thanks for fixing my small nits!

**Pull request merged**